README.md

```diff
@@ -109,6 +109,17 @@ Participation in the Kubernetes community is governed by the [Kubernetes Code of
 
    ***NOTE*** The private key used to connect to the master node is by default (currently hardcoded) at ~/.ssh/id_ibmcloud, so make sure you copy the generated key's private key info into this file.
 
+   #### Quick notes on machines.yaml
```

Comment:
Please run `doctoc`

Reply:
I prefer to migrate those info into a tips or cloudinfo helper file in doc subfolder...